tests/marathon_lb/test_stats.py

```diff
@@ -0,0 +1,52 @@
+# stdlib imports
```

Comment:
Whilst not *really* related to the main subject of this PR, the tests that covered this module until now lived in `tests/test_bluegreen.py` and they've just been moved here to a more appropriate location as part of getting rid of that file.